index.html

```diff
@@ -274,7 +274,7 @@ <h3 class="feature-title">Contribute</h3>
                                     </div>
                                     <h3 class="feature-title">Support</h3>
                                     <p class="text-sm">We're always wanting to add new spoken and programming languages to our repository. Please lege-<u>share</u> our mission.
-                                    </br></br>From beginners learning how to code to collaborating with diverse teams around the world, Legesher connects us all.</p>
+                                    </br></br>From beginners learning how to code to collaborate with diverse teams around the world, Legesher connects us all.</p>
```

Comment:
To be honest, I'm not really crazy about this sentence in the first place. I've been thinking of another way to express the different use cases, but haven't found anything that really sticks. would love your thoughts!

Reply:
Made the required changes @madiedgar